operator/controllers/model_initializer_injector.go

```diff
@@ -95,11 +95,13 @@ func getStorageInitializerConfigsFromMap(configMap *corev1.ConfigMap) (*StorageI
 			panic(fmt.Errorf("Unable to unmarshall %v json string due to %v ", StorageInitializerConfigMapKeyName, err))
 		}
 	}
-	//Ensure that we set proper values for CPU/Memory Limit/Request
-	resourceDefaults := []string{storageInitializerConfig.MemoryRequest,
+	// Ensure that we set proper values for CPU/Memory Limit/Request
+	resourceDefaults := []string{
```

Comment:
Why are there fmt changes? Is there a version disparity perhaps?

Reply:
The suggested changes on format look good but best to be added on a separate PR and focus this one on the inject storage initialiser logic